setup.py

```diff
@@ -25,14 +22,18 @@
     install_requires=["pytest"],
     extras_require={"testing": tests_require},
     classifiers=[
-        "Development Status :: 1 - Planning",
+        "Development Status :: 4 - Beta"
         "Intended Audience :: Developers",
         "Operating System :: OS Independent",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
```

Comment:
Let's add these only when they are tested in the CI

Reply:
I'll add the workflows to this PR :)